src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+class Calculator extends React.Component{
+    render(){
+        return(
+            <div className="mainContainer">
+            <div className="buttonWrapper">
+                <div className="outputScreen">0</div>
+                <div className="buttonClick restBtn functBtn">AC</div>
+                <div className="buttonClick plus_minusBtn functBtn">+/-</div>
+                <div className="buttonClick percentBtn functBtn">%</div>
+                <div className="buttonClick divisionBtn extraBtn">+</div>
```

Comment:
- [ ] (OPTIONAL) Kindly correct your division (`÷`)  and multiplication (`×`) symbols 💯

Reply:
Ok, I will definitely do that. Thanks